dev/planning-example.py

```diff
@@ -0,0 +1,292 @@
+
+# It's easy to construct date/times from the observer, since it knows about
+# its timezone.  These dates can be passed directly to target check methods.
+
+# dates default to midnight if not otherwise specified
+# Date can be anything that astropy.time.Time will accept
+day = obs.get_date("2015-05-01")
+
+# Can get more and more specific
+t1 = obs.get_date("2015-05-01 21:00")
+t1 = obs.get_date("2015-05-01 21:33:41")
+
+# default is current time
+cur_time = obs.get_date()
+
+# Dates are assumed to be in the observer's time zone unless a time zone
+# specifier is given
+t2_utc = obs.get_date("2015-05-01 21:33:41 UTC")
+
+"""
+We can ask about typical times of interest at this observing position
+Returned dates are assumed to be in the observer's time zone for convenience.
+The date parameter, if given, can be an astropy Time or a datetime instance.
+If no parameter is given, the current date/time is assumed.
+"""
+
+# sunset
+obs.next_sunset(date=day)
+obs.previous_sunset(date=day)
+
+# sunrise
+obs.next_sunrise(date=day)
+obs.previous_sunrise(date=day)
+
+# moon rise
+obs.next_moon_rise(date=day)
+obs.previous_moon_rise(date=day)
+
+# moon set
+obs.next_moon_set(date=day)
+obs.previous_moon_set(date=day)
+
+# The above functions can be called with an `angle` keyword argument to specify
+# a particular horizon angle for rising or setting, or can be called with
+# convenience functions for particular morning/evening twilight:
+
+obs.next_sunset(date=day, angle=18*u.degree) # (i.e., astronomical twilight)
+
+# evening (astronomical) twilight
+obs.evening_astronomical(date=day)
+
+# evening (nautical) twilight
+obs.evening_nautical(date=day)
+
+# evening (civil) twilight
+obs.evening_civil(date=day)
+
+# morning (nautical) twilight
+obs.morning_nautical(date=day)
+
+# morning (civil) twilight
+obs.morning_civil(date=day)
+
+# morning (astronomical) twilight
+obs.morning_astronomical(date=day)
+
+# what is the moon illumination?
+# returns a float, which is percentage of the moon illuminated
+obs.moon_illumination(date=t1)
+
+# what is the moon altitude and azimuth?
+# returns an altaz coordinate
+obs.moon_position(date=t1)
+
+# Other sun-related convenience functions:
+obs.noon(date=day)
+obs.midnight(date=day)
+
+# ==============
+# Target objects
+# ==============
+'''
+A target object defines a single observation target, with coordinates
+and an optional name.
+'''
+from astroplan import FixedTarget, airmass_plot
+
+# Define a target.
+t1 = FixedTarget(name='Polaris', ra='02:31:49.09', dec='+89:15:50.8')
+
+# initializing from astropy entities directly
+from astropy.coordinates import SkyCoord
+
+t1 = FixedTarget(name='Polaris',
+               coord=SkyCoord('02h31m49.09s', '+89d15m50.8s', frame='icrs'))
+
+# Leaves scope for NonSiderealTarget, etc.
+
+# Convenience methods for looking up/constructing targets by name via
+# astroquery?
+
+# Plot the airmass for the specified target:
+airmass_plot(t1, observer=obs, datetime=day)
```

Comment:
@cdeil Good point.  I think we had decided I would be producing plotting funcs/sample plots at some point.  Here's what I have from our first meeting (since accepted to GSoC):

>Jazmin: high-level use of tools, test cases; possibly producing plots/planning via web browser (ipython notebook?)

and discussion with @eteq 

>Write more astropy-friendly version of JPL ephemerides package used in skyfield?
>https://pypi.python.org/pypi/jplephem 
>Jazmin will spend time putting this together in first phase of coding

I'm currently looking at building the docs for a mock API, and once the discussion here winds down/I figure out Sphinx, I'll put up a PR with what I've got for the class/methods hierarchy so far.